# Translations template for manila.
# Copyright (C) 2015 ORGANIZATION
# This file is distributed under the same license as the manila project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2015.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: manila 2015.1.dev63\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2015-01-07 06:12+0000\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 1.3\n"

#: manila/exception.py:94
msgid "Exception in string format operation."
msgstr ""

#: manila/exception.py:96
#, python-format
msgid "%(name)s: %(value)s"
msgstr ""

#: manila/manager.py:163
#, python-format
msgid "Error during %(full_task_name)s: %(e)s"
msgstr ""

#: manila/quota.py:983
#, python-format
msgid "Failed to commit reservations %s"
msgstr ""

#: manila/quota.py:1007
#, python-format
msgid "Failed to roll back reservations %s"
msgstr ""

#: manila/service.py:237 manila/openstack/common/service.py:269
msgid "Unhandled exception"
msgstr ""

#: manila/service.py:508
msgid "Recovered model server connection!"
msgstr ""

#: manila/service.py:514
msgid "model server went away"
msgstr ""

#: manila/api/extensions.py:246
#, python-format
msgid "Exception loading extension: %s"
msgstr ""

#: manila/api/middleware/fault.py:46
#, python-format
msgid "Caught error: %s"
msgstr ""

#: manila/api/openstack/wsgi.py:583
#, python-format
msgid "Exception handling resource: %s"
msgstr ""

#: manila/api/v1/share_networks.py:122
msgid "Failed to update usages deleting share-network."
msgstr ""

#: manila/db/migrations/alembic/versions/162a3e673105_manila_init.py:403
msgid "Exception while creating table."
msgstr ""

#: manila/db/sqlalchemy/api.py:2120
#, python-format
msgid "VolumeType %s deletion failed, VolumeType in use."
msgstr ""

#: manila/network/linux/interface.py:175 manila/network/linux/interface.py:217
#, python-format
msgid "Failed unplugging interface '%s'"
msgstr ""

#: manila/network/linux/ovs_lib.py:48
#, python-format
msgid "Unable to execute %(cmd)s. Exception: %(exception)s"
msgstr ""

#: manila/network/neutron/api.py:167
#, python-format
msgid "Neutron error creating port on network %s"
msgstr ""

#: manila/openstack/common/loopingcall.py:95
msgid "in fixed duration looping call"
msgstr ""

#: manila/openstack/common/loopingcall.py:138
msgid "in dynamic looping call"
msgstr ""

#: manila/openstack/common/policy.py:535 manila/openstack/common/policy.py:815
#, python-format
msgid "Failed to understand rule %s"
msgstr ""

#: manila/openstack/common/policy.py:545
#, python-format
msgid "No handler for matches of kind %s"
msgstr ""

#: manila/scheduler/filter_scheduler.py:212
#, python-format
msgid ""
"Error scheduling %(share_id)s from last share-service: %(last_host)s : "
"%(exc)s"
msgstr ""

#: manila/scheduler/scheduler_options.py:68
#, python-format
msgid "Could not stat scheduler options file %(filename)s: '%(e)s'"
msgstr ""

#: manila/scheduler/scheduler_options.py:78
#, python-format
msgid "Could not decode scheduler options: '%(e)s'"
msgstr ""

#: manila/scheduler/filters/capacity_filter.py:39
msgid "Free capacity not set: volume node info collection broken."
msgstr ""

#: manila/share/api.py:223
msgid "Failed to update quota for deleting share"
msgstr ""

#: manila/share/driver.py:140
#, python-format
msgid "Recovering from a failed execute. Try number %s"
msgstr ""

#: manila/share/manager.py:91
#, python-format
msgid ""
"Caught exception trying ensure share '%(s_id)s'. Exception: \n"
"%(e)s."
msgstr ""

#: manila/share/manager.py:108
#, python-format
msgid ""
"Unexpected exception during share access allow operation. Share id is "
"'%(s_id)s', access rule type is '%(ar_type)s', access rule id is "
"'%(ar_id)s', exception is '%(e)s'."
msgstr ""

#: manila/share/manager.py:217
#, python-format
msgid "Share server %s does not exist."
msgstr ""

#: manila/share/manager.py:227
msgid "Failed to get share server for share creation."
msgstr ""

#: manila/share/manager.py:246
#, python-format
msgid "Share %s failed on creation."
msgstr ""

#: manila/share/manager.py:292
msgid "Failed to update usages deleting share"
msgstr ""

#: manila/share/manager.py:366
msgid "Failed to update usages deleting snapshot"
msgstr ""

#: manila/share/manager.py:507
#, python-format
msgid "Share server %s failed on deletion."
msgstr ""

#: manila/share/volume_types.py:40
#, python-format
msgid "DB error: %s"
msgstr ""

#: manila/share/volume_types.py:126
#, python-format
msgid ""
"Default volume type is not found, please check default_volume_type "
"config: %s"
msgstr ""

#: manila/share/drivers/generic.py:231
#, python-format
msgid "Failed to sync mount files on server '%s'."
msgstr ""

#: manila/share/drivers/generic.py:238
#, python-format
msgid "Failed to mount all shares on server '%s'."
msgstr ""

#: manila/share/drivers/glusterfs.py:137
#, python-format
msgid "Error retrieving volume info: %s"
msgstr ""

#: manila/share/drivers/glusterfs.py:162
#, python-format
msgid "Error in disabling access to the entire GlusterFS volume: %s"
msgstr ""

#: manila/share/drivers/glusterfs.py:173
#, python-format
msgid "Error in enabling creation of shares of specific size: %s"
msgstr ""

#: manila/share/drivers/glusterfs.py:233
#, python-format
msgid "Could not mount the Gluster volume %s"
msgstr ""

#: manila/share/drivers/glusterfs.py:303
#, python-format
msgid "Unable to create share %s"
msgstr ""

#: manila/share/drivers/glusterfs.py:317
#, python-format
msgid ""
"Cannot cleanup share, %s, that errored out during its creation, but "
"exists in GlusterFS volume."
msgstr ""

#: manila/share/drivers/glusterfs.py:329
#, python-format
msgid "Unable to delete share %s"
msgstr ""

#: manila/share/drivers/glusterfs.py:388
#, python-format
msgid "Error in gluster volume set: %s"
msgstr ""

#: manila/share/drivers/emc/plugins/vnx/connection.py:569
#, python-format
msgid "Could not setup server. Reason: %s."
msgstr ""

#: manila/share/drivers/emc/plugins/vnx/connection.py:691
#, python-format
msgid "Could not find CIFS server by name: %s."
msgstr ""

#: manila/share/drivers/emc/plugins/vnx/connection.py:736
msgid "Could not get storage pool list."
msgstr ""

#: manila/share/drivers/emc/plugins/vnx/connection.py:863
#, python-format
msgid "Could not get mount point. Reason: %s."
msgstr ""

#: manila/share/drivers/emc/plugins/vnx/helper.py:89
#, python-format
msgid ""
"REQ: [%(method)s] %(url)s %(req_hdrs)s\n"
"REQ BODY: %(req_b)s\n"
"RESP: [%(code)s] %(resp_hdrs)s\n"
"RESP BODY: %(resp_b)s\n"
msgstr ""

#: manila/share/drivers/emc/plugins/vnx/helper.py:1159
#, python-format
msgid "Error running SSH command: %s"
msgstr ""

#: manila/share/drivers/ganesha/manager.py:191
#, python-format
msgid "Error while executing management command on Ganesha node %(tag)s: %(msg)s."
msgstr ""

#: manila/share/drivers/ganesha/manager.py:330
#, python-format
msgid "Invalid export database on Ganesha node %(tag)s: %(db)s."
msgstr ""

#: manila/share/drivers/ibm/gpfs.py:696
#, python-format
msgid "Invalid metadata %(attr)s for share %(share)s."
msgstr ""

#: manila/share/drivers/netapp/cluster_mode.py:170
#, python-format
msgid "Could not get licenses list. %s."
msgstr ""

#: manila/share/drivers/netapp/cluster_mode.py:415
msgid "Failed to create network interface"
msgstr ""

#: manila/share/drivers/netapp/cluster_mode.py:496
msgid "DNS exists for vserver."
msgstr ""

#: manila/share/drivers/netapp/cluster_mode.py:827
#, python-format
msgid "Vserver %s does not exist."
msgstr ""

#: manila/share/drivers/netapp/cluster_mode.py:838
#, python-format
msgid "Volume %s is already offline."
msgstr ""

#: manila/share/drivers/netapp/cluster_mode.py:862
#, python-format
msgid "CIFS server does not exist for vserver %s"
msgstr ""

#: manila/share/drivers/netapp/cluster_mode.py:1190
#, python-format
msgid "User %s does not exist."
msgstr ""

#: manila/share/drivers/netapp/cluster_mode.py:1192
#, python-format
msgid "Rule %s does not exist."
msgstr ""

#: manila/share/drivers/zfssa/zfssarest.py:237
#, python-format
msgid ""
"Error Deleting Share: %(share)s to Pool: %(pool)s Project: %(project)s  "
"Return code: %(ret.status)d Message: %(ret.data)s."
msgstr ""

#: manila/share/drivers/zfssa/zfssashare.py:266
#, python-format
msgid "snapshot %s: has clones"
msgstr ""

